spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
I'm running Ruby 2.0 on OSX.

To be clear my testing was your new spec example by itself, run `rspec` as normal: it passes. Then run the test with ascii as the default external encoding with that command I posted: error from spec. In neither case do I get an error from `ConfigLoader`'s implementation.